### DISCUSSION TITLE
drop collection usage

### DISCUSSION THREAD

**Discussion by vihariazure** [milvus-io/milvus#38480] [Q&A and General discussion]
my requirement is to delete data inside the collection 

for above purpose do i need to use (drop collection vs delete entities )

for drop collection which code do i need to use
from pymilvus import utility
utility.drop_collection("book")

(or)
client.drop_collection(collection_name="book")

**Comment by yhmo** [comment]
If you want to delete all the entities of a collection, it is better to drop the collection.

client.drop_collection is preferred.

**Reply by vihariazure** [reply]
thanks for the reply
